frontend/src/queries/usePltTokenQuery.ts

```diff
@@ -28,6 +28,7 @@ const PLT_TOKEN_QUERY = gql<PltTokenQueryResponse>`
 					asString
 				}
 				totalUniqueHolders
+				normalizedCurrentSupply
```

Comment:
Update the changelog, especially since it expects a new version of the backend to be released first